main.py

```diff
@@ -1,51 +1,30 @@
-
-MAJOR_COLORS = ['White', 'Red', 'Black', 'Yellow', 'Violet']
-MINOR_COLORS = ["Blue", "Orange", "Green", "Brown", "Slate"]
-
+from colorCode import ColorCode
+global colorCode_Obj
 
 def color_pair_to_string(major_color, minor_color):
-  return f'{major_color} {minor_color}'
-
-
-def get_color_from_pair_number(pair_number):
-  zero_based_pair_number = pair_number - 1
-  major_index = zero_based_pair_number // len(MINOR_COLORS)
-  if major_index >= len(MAJOR_COLORS):
-    raise Exception('Major index out of range')
-  minor_index = zero_based_pair_number % len(MINOR_COLORS)
-  if minor_index >= len(MINOR_COLORS):
-    raise Exception('Minor index out of range')
-  return MAJOR_COLORS[major_index], MINOR_COLORS[minor_index]
+    return f'{major_color} {minor_color}'
 
+def color_coding_manual():
```

Comment:
This method is part of production code. keep production code and test code separate.

Reply:
Hi @prithamvshetty, I have modified the scripts and kept this method inside a different colorCodingManual.py which is printing the required color code of wire and its number